ui-playground/lib/background.js

```diff
@@ -0,0 +1,114 @@
+/* globals chrome:false */
+
+'use strict';
+
+chrome.browserAction.setBadgeText({text: '🚲ᕕ( ᐛ)ᕗ'});
+// chrome.browserAction.setBadgeBackgroundColor({color: '#FFF'});
+
+var chillOut = function (aTab) {
```

Comment:
Maybe just me, but I found this function quite hard to read. As I understand it, it's doing this:
* if the active tab has "chilloutand..." loaded already, do nothing
* if some other tab has "chilloutand..." loaded already, switch to it
* otherwise, open a new tab with "chilloutand..." loaded

To me, code like this would be easier to understand:

    var chillOut = function (aTab) {
      console.log("Chilling…");
      var catGifs = 'http://chilloutandwatchsomecatgifs.com/';

      chrome.tabs.query({'url': catGifs}, (tabs) => {
        if (tabs.length) {
          for (var tab in tabs) {
            if (tab.active) {
              return;
            }
          }
          chrome.tabs.update(tabs[0].id, {'active': true});
        }
        else {
          chrome.tabs.create({'url': catGifs, 'active': true}, (tab) => {
            console.log(tab);
          });  
        }
      });
    }

But maybe this is a just a personal thing, or I'm missing something.